tests/Microplate/Scalars/Column12Test.php

```diff
@@ -23,20 +23,20 @@ public function testSerializeThrowsIfColumn96WellIsNotAnInt(): void
         $this->expectException(\InvalidArgumentException::class);
         $this->expectExceptionMessage('Value not in range: "12".');
 
-        (new Column96Well())->serialize('12');
+        (new Column12())->serialize('12');
     }
 
     public function testSerializeThrowsIfColumn96WellIsInvalid(): void
     {
         $this->expectException(\InvalidArgumentException::class);
         $this->expectExceptionMessage('Value not in range: 13.');
 
-        (new Column96Well())->serialize(13);
+        (new Column12())->serialize(13);
     }
 
     public function testSerializePassesWhenColumn96WellIsValid(): void
```

Comment:
Namen der Tests sind inakkurat

```suggestion
    public function testSerializeValid(): void
```